examples/multiomics_mogonet/main.py

```diff
@@ -0,0 +1,107 @@
+"""
+Multi-omics Integration via Graph Convolutional Networks for Biomedical Data Classification.
```

Comment:
I think we have agreed to use multiomics instead of multi-omics. Could you please check all to fix so that I do not have to repeat the same comment?

Reply:
Is this good for general biomedical data? Please be specific if not.